eras/conway/impl/src/Cardano/Ledger/Conway/Core.hs

```diff
@@ -0,0 +1,4 @@
+module Cardano.Ledger.Conway.Core
```

Comment:
Not sure about this, does it bring any value at this point, or better to add it when needed? I don't necessarily mind, just wondering

Reply:
I actually didn't mean to push this file